### ISSUE TITLE
Does he-tree recognize a drop on ?

### ISSUE THREAD

**Issue by duderion** [phphe/he-tree-vue#44]
Hi phphe,

thanks for your great tree library, i bought the pro version.

my question is: 
is the he tree able to recognize if i drop a node (from another tree) directly onto a node in the other tree ?
if yes, how can that be done ?

Thanks in advance
Adrian

**Comment by phphe** [comment]
yes. The node can be moved freely between multiple trees with pro plugin.
